src/index.js

```diff
@@ -9,12 +9,16 @@ import { createStore } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import rootReducer from './Reducers';
 
+const store = createStore(rootReducer, composeWithDevTools())
+
+
+
 ReactDOM.render(
-  <Router>
-    <React.StrictMode>
```

Comment:
after all that... no strict mode?

Reply:
https://reactjs.org/docs/strict-mode.html
not sure why it needed not to much experience with it.